tests/unit/flow/test_flow_visualization.py

```diff
@@ -0,0 +1,28 @@
+import os
+import numpy as np
+from PIL import Image
+from jina.flow import Flow
+
+
+def test_visualization_URL():
+    cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+    flow = (Flow().add(name='pod_a')
+            .add(name='pod_b', needs='gateway')
+            .join(needs=['pod_a', 'pod_b']))
+    url = flow.plot()
+    expected_text = 'https://mermaidjs.github.io/mermaid-live-editor/#/view/Z3JhcGggVEQKZ2F0ZXdheVtnYXRld2F5XSAtLT4gcG9kX2FbcG9kX2FdCmdhdGV3YXlbZ2F0ZXdheV0gLS0+IHBvZF9iW3BvZF9iXQpwb2RfYVtwb2RfYV0gLS0+IGpvaW5lcltqb2luZXJdCnBvZF9iW3BvZF9iXSAtLT4gam9pbmVyW2pvaW5lcl0='
```

Comment:
```suggestion
    expected_text = 'https://mermaidjs.github.io/mermaid-live-editor/#/view/Z3JhcGggVEQKZ2F0ZXdheVtnYXRld2F5XSAtLT4gcG9kX2FbcG9kX2FdCmdhdGV3YXlbZ2F0ZXdheV0gLS0+IHBvZF9iW3BvZF9iXQpwb2RfYVtwb2RfYV0gLS0+IGpvaW5lcltqb2luZXJdCnBvZF9iW3BvZF9iXSAtLT4gam9pbmVyW2pvaW5lcl0='
```

This test is probably too harsh. You can just check what followed by `https://mermaidjs.github.io/mermaid-live-editor/#/view/` is non-empty